src/components/utils/hooks/useGetConversation.tsx

```diff
@@ -44,7 +44,7 @@ const useGetConversation = (conversationKey: string, profile?: Profile) => {
     };
     createNewConversation();
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [profile]);
+  }, [profile, conversations]);
```

Comment:
This is going to re-run any time any conversation changes. Should probably be scoped to the single conversation we care about.

Reply:
Glad you called this out! Updated to only listen for the `selectedConversation`!